changelog/1367.feature.rst

```diff
@@ -0,0 +1,26 @@
+**Support for subtests** has been added.
+
+:ref:`subtests <subtests>` are an alternative to parametrization, useful in situations where the parametrization values are not all known at collection time.
+
```

Comment:
Perhaps should mention that it integrates/replaces `pytest-subtests` with some differences?